components/bl-multiple-bar-chart-component/src/index.js

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+
+import { validate } from './validate';
+import { Bar } from './bar';
+
+const { cn } = BackendlessUI.CSSUtils;
+const percentageMarksList = [0, 25, 50, 75, 100];
```

Comment:
Could we add property for that? Also, should be a variant without marks(if we not provide them)